tests/e2e/completions/path.rs

```diff
@@ -79,3 +79,29 @@ fn multi_segment_path() {
     completion_label = "Baz"
     "#);
 }
+
+#[test]
+fn multi_segment_path_partial() {
+    test_transform!(test_completions_text_edits,"
+    mod foo {
+        pub mod bar {
+            pub struct Baz {}
+        }
+    }
+
+    fn a() {
+        bar::B<caret>
+    }
```

Comment:
```suggestion
    mod foo {
        pub mod bar {
            pub struct Baz {}
        }
        pub struct Boo {}
    }

    fn a() {
        bar::B<caret>
    }
```
just to check if it is not catched